.github/workflows/ts_test.yaml

```diff
@@ -48,5 +48,34 @@ jobs:
       - name: Generate Proof
         run: nargo prove --package main --oracle-resolver=http://localhost:5555
 
+      - name: Run nargo codegen-verifier
+        run: nargo codegen-verifier --package main
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
```

Comment:
We should use consistent versions in all CI scripts. Please switch to specific pinned nightly version